bm25s/__init__.py

```diff
@@ -876,8 +876,8 @@ def save(
         # Save the vocab dictionary
         vocab_path = save_dir / vocab_name
 
-        with open(vocab_path, "w") as f:
-            f.write(json_functions.dumps(self.vocab_dict))
+        with open(vocab_path, "wt", encoding='utf-8') as f:  
+            json.dump(self.vocab_dict, f, ensure_ascii=False)
```

Comment:
Could you make the changes directly to json_functions, such that it accepts a kwargs? This way we can keep using orjson when possible.

I'm also curious if orjson supports ensure_ascii... Do you have any experience?

Reply:
@xhluca In orjson, the default behavior already aligns with ensure_ascii=False in Python's built-in json.dump and json.dumps. I've modified the json_functions file to take kwargs also. I've tested it with non ASCII characters for which it was throwing errors earlier. Please review.